doc/manual/src/command-ref/nix-channel.md

```diff
@@ -1,3 +1,16 @@
+> **Note**
+>
+> Channels are deprecated, because they are not portable across machines and therefore not reproducible.
+>
+> We recommend declaring dependencies explicitly using:
+> - [`fetchurl`], [`fetchTarball`], or [`fetchGit`] in Nix expressions
+> - the [`-I` option] in command line invocations
+
+[`fetchurl`]: ../language/builtins.md#builtins-fetchurl
```

Comment:
We really need to stop with the atrocious "footnote" style of linking. Just write `[link](url)`, which is much easier to maintain.